modules/webhook-configuring-acs.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-generic-webhooks.adoc
+:_module-type: PROCEDURE
+[id="webhook-configuring-acs_{context}"]
+= Configuring {product-title}
+
+[role="_abstract"]
+Create a new integration in {product-title} by using the webhook URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
+. Enter a name for *Integration Name*.
+. Enter the webhook URL in the *Endpoint* field.
+. If your webhook receiver is using an untrusted certificate, enter a CA certificate in the *CA Cert* box, otherwise leave it blank.
```

Comment:
s/box, otherwise/box; otherwise,

Reply:
Thank you @bobfuru 
Sorry but I don't understand the comment.
I've change it to the following:
```
. If your webhook receiver uses an untrusted certificate, enter a CA certificate in the *CA Cert* field, otherwise leave it blank.
```

Reply:
I was suggesting a small restructuring of punctuation ("box" or "field" both seem fine).
Instead of this:
```
...field, otherwise leave it blank.
```
Use this:
```
...field; otherwise, leave it blank.
```
I should have included a citation for clarity. Here is ISG's guidance:
> Use a semicolon between independent clauses that are connected by a conjunctive adverb or a transitional phrase.
Conjunctive adverbs include also, besides, consequently, conversely, finally, however, instead, likewise, next, otherwise, similarly, specifically, still, subsequently, then, and therefore.